src/main/java/taxi/model/User.java

```diff
@@ -0,0 +1,70 @@
+package taxi.model;
+
+import java.util.Objects;
+
+public class User {
```

Comment:
this is a redundant model, you should modify `Driver` and use `Driver` instead of `User`